### PR TITLE
fix(rdb): block low latency

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**
added 'blocw low latency'

### CODE REVIEW

File: managed-databases/postgresql-and-mysql/concepts.mdx [hunk 1]
```diff
@@ -31,21 +31,39 @@ Database Instances have settings that allow you to tune the behavior of their da
...
+
+Basic Block Storage is a volume type. With this type your storage is decoupled from your compute resources. You can increase your storage space without changing your node type. You can define your storage capacity by entering the desired amount in the box.
+
+</Concept>
+
+<Concept opened>
+
+## Block Storage Low Latency
+
+Low Latency is a storage type that works like the [Basic Block Storage](#basic-block-storage), while providing enhanced efficiency. You can choose between **5k** or **15k IOPS** according to your workloads input/output operations needs, with lower latency and high resiliency. You can increase your volume to up to 10 TB.
```
Comment (abarbare) [inline managed-databases/postgresql-and-mysql/concepts.mdx:44]:
```suggestion
Block Storage Low Latency is a storage type that works like the [Basic Block Storage](#basic-block-storage), while providing enhanced efficiency. You can choose between **5k** or **15k IOPS** according to your workloads input/output operations needs, with lower latency and high resiliency. You can increase your volume to up to 10 TB.
```

File: managed-databases/postgresql-and-mysql/concepts.mdx [hunk 2]
```diff
@@ -31,21 +31,39 @@ Database Instances have settings that allow you to tune the behavior of their da
 
 <Concept opened>
 
+## Basic Block Storage
+
+Basic Block Storage is a volume type. With this type your storage is decoupled from your compute resources. You can increase your storage space without changing your node type. You can define your storage capacity by entering the desired amount in the box.
```
Comment (abarbare) [inline managed-databases/postgresql-and-mysql/concepts.mdx:36]: do we want to add that the max size is also 10T here as well?

File: managed-databases/postgresql-and-mysql/concepts.mdx (hunk 1 above)
Comment (abarbare) [reply managed-databases/postgresql-and-mysql/concepts.mdx:44]: 15k is not yet available. Please wait a bit so we can indicate it or not for the beta

File: managed-databases/postgresql-and-mysql/concepts.mdx [hunk 3]
```diff
@@ -134,3 +164,13 @@ A Read Replica is a live copy of a Database Instance that behaves like an Instan
...
 
 </Concept>
+
+<Concept>
+
+## Volume type
+
+This is the type of your storage used on your Database Instances. Three types are available: [Local](#local-storage), [Basic Block](#basic-block-storage) and [Block Low Latency](#block-storage-low-latency).
+
+You can select the volume type upon Database Instance creation, and you can [change the type](/managed-databases/postgresql-and-mysql/how-to/change-volume-type) anytime.
```
Comment (abarbare) [inline managed-databases/postgresql-and-mysql/concepts.mdx:174]: Low Latency block storage is only available on second generation hardware

File: managed-databases/postgresql-and-mysql/how-to/change-volume-type.mdx [hunk 4]
```diff
@@ -37,10 +37,8 @@ It is possible to change your volume type from local to Block Storage with the f
 
     The Database Instance goes into the `Initializing` state.
 
-    Alternatively, you can change the volume type from Block Storage to local storage by following the same steps.
-
     <Message type="important">
-      Make sure your current node type has a local storage size greater than or equal to your current Block Storage size. If this is not the case, you must [upgrade your current node type](/managed-databases/postgresql-and-mysql/how-to/upgrade-a-database/) to one with enough local storage before you change the volume type.
+      If you are changing from a Block to a local volume, make sure your current node type has a local storage size greater than or equal to your current Block Storage size first. If this is not the case, you must [upgrade your current node type](/managed-databases/postgresql-and-mysql/how-to/upgrade-a-database/) to one with enough local storage before you change the volume type.
```
Comment (abarbare) [inline managed-databases/postgresql-and-mysql/how-to/change-volume-type.mdx:41]:
```suggestion
      If you are changing from a Block to a local volume, make sure to select a node with  a local storage size greater than or equal to your current Block Storage.
```

File: managed-databases/postgresql-and-mysql/how-to/clone-a-database-instance.mdx [hunk 5]
```diff
@@ -37,7 +37,9 @@ The clone feature allows you to create a new Database Instance from an existing
 5. Enter a name for your database or leave the default name suggested by the Scaleway console.
 6. Select a node type. You can keep the same node type, or upgrade to a bigger one.
     <Message type="important">
-      If you have a Database Instance that uses a local volume, and you want to upgrade to an Instance that uses Block volumes, you must [change the volume type](/managed-databases/postgresql-and-mysql/how-to/change-volume-type) to Block storage first.
+      If you have a Database Instance that uses a local volume, and you want to upgrade to an Instance that uses Block volumes, you must [change the volume type](/managed-databases/postgresql-and-mysql/how-to/change-volume-type) to Block storage first. <br /><br />
```
Comment (abarbare) [inline managed-databases/postgresql-and-mysql/how-to/clone-a-database-instance.mdx:40]:
```suggestion
      If you have a new-generation Database Instance using a local volume Node Type and want to upgrade to an Instance that uses Block volumes Node Type, you must [change the volume type](/managed-databases/postgresql-and-mysql/how-to/change-volume-type) to Block storage first. <br /><br />
```

File: managed-databases/postgresql-and-mysql/how-to/create-a-database.mdx [hunk 6]
```diff
@@ -40,13 +40,15 @@ Compared to traditional database management, which requires customers to provide
         </Message>
     - Select a node type.
     - Choose your storage type and capacity. Two types of storage are available:
+        - **Basic Block Storage** - With this type, your storage is decoupled from your compute resources. You can increase your storage space without changing your node type. You can define your storage capacity by entering the desired amount in the box.
```
Comment (abarbare) [inline managed-databases/postgresql-and-mysql/how-to/create-a-database.mdx:43]: See my previous comment about the max size

File: managed-databases/postgresql-and-mysql/how-to/create-a-database.mdx [hunk 7]
```diff
@@ -40,13 +40,15 @@ Compared to traditional database management, which requires customers to provide
...
+
+        - **Block Storage Low Latency**: This storage type works like the Basic Block Storage, while providing enhanced efficiency. You can choose between **5k** or **15k IOPS** according to your workloads input/output operations needs, with lower latency and high resiliency. You can increase your volume to up to 10 TB.
```
Comment (abarbare) [inline managed-databases/postgresql-and-mysql/how-to/create-a-database.mdx:45]: Same comment as before for 15k

File: managed-databases/postgresql-and-mysql/how-to/create-a-database.mdx [hunk 8]
```diff
@@ -40,13 +40,15 @@ Compared to traditional database management, which requires customers to provide
...
+
         - **Local SSD Storage** - Your storage is fixed and tied to your compute resource.
         <Message type="note">
           The local storage option is only available with the first generation node types.
         </Message>
-
-        - **Block SSD Storage** - With this type, your storage is decoupled from your compute resources. You can increase your storage space without changing your node type, with no downtime. You can define your storage capacity by entering the desired amount in the box. You can increase your volume to up to 10 TB.
-    - Enable automatic backups, if necessary.
+    - Select the snapshot configuration. You can either leave the default **Automatic** option to enable [autobackups](/managed-databases/postgresql-and-mysql/how-to/enable-autobackup), or select [Manual snapshots](/managed-databases/postgresql-and-mysql/how-to/manage-manual-backups).
```
Comment (abarbare) [inline managed-databases/postgresql-and-mysql/how-to/create-a-database.mdx:51]: the last link point out on a link with logical backups doc whereas we want to explain how to perform manual snapshots which is not the same

Comment (abarbare) [reply managed-databases/postgresql-and-mysql/how-to/create-a-database.mdx:51]:
```suggestion
    - Select the snapshot configuration. You can either leave the default **Automatic** option to enable [autobackups](/managed-databases/postgresql-and-mysql/how-to/enable-autobackup), or select [Manual snapshots](/managed-databases/postgresql-and-mysql/how-to/manage-snapshots).
```

File: managed-databases/postgresql-and-mysql/how-to/upgrade-a-database.mdx [hunk 9]
```diff
@@ -36,7 +36,9 @@ You can upgrade your Database Instances in two ways: by increasing its Block Sto
 ## How to change the node type
 
 <Message type="important">
-  If you have a Database Instance that uses a local volume, and you want to upgrade to an Instance that uses Block volumes, you must [change the volume type](/managed-databases/postgresql-and-mysql/how-to/change-volume-type) to Block storage first.
+  If you have a Database Instance that uses a local volume, and you want to upgrade to an Instance that uses Block volumes, you must [change the volume type](/managed-databases/postgresql-and-mysql/how-to/change-volume-type) to Block storage first. <br /><br />
```
Comment (abarbare) [inline managed-databases/postgresql-and-mysql/how-to/upgrade-a-database.mdx:39]:
```suggestion
  If you have a Database Instance that uses a local volume, and you want to upgrade to a Node Type that uses Block volumes, you must [change the volume type](/managed-databases/postgresql-and-mysql/how-to/change-volume-type) to Block storage first. <br /><br />
```

File: managed-databases/postgresql-and-mysql/how-to/upgrade-a-database.mdx [hunk 10]
```diff
@@ -36,7 +36,9 @@ You can upgrade your Database Instances in two ways: by increasing its Block Sto
...
+
+  Similarly, if you have a new-generation Database Instance using a [Low Latency Block volume](/managed-databases/postgresql-and-mysql/concepts/#block-storage-low-latency) and want to switch to a first-generation Instance, you must first change the volume type to [Basic Block Storage](/managed-databases/postgresql-and-mysql/concepts/#basic-block-storage).
```
Comment (abarbare) [inline managed-databases/postgresql-and-mysql/how-to/upgrade-a-database.mdx:41]:
```suggestion
  Similarly, if you have a new-generation Database Instance using a [Low Latency Block volume](/managed-databases/postgresql-and-mysql/concepts/#block-storage-low-latency) Node Type and want to switch to a first-generation Node Type, you must first change the volume type to [Basic Block Storage](/managed-databases/postgresql-and-mysql/concepts/#basic-block-storage).
```

File: managed-databases/postgresql-and-mysql/concepts.mdx [hunk 11]
```diff
@@ -31,21 +31,39 @@ Database Instances have settings that allow you to tune the behavior of their da
 
 <Concept opened>
 
+## Basic Block Storage
+
+Basic Block Storage is a volume type. With this type your storage is decoupled from your compute resources. You can increase your storage space without changing your node type. You can define your storage capacity by entering the desired amount in the box. You can increase your volume to up to 10 TB.
```
Comment (ldecarvalho-doc) [inline managed-databases/postgresql-and-mysql/concepts.mdx:36]:
```suggestion
Basic Block Storage is a volume type. With this type your storage is decoupled from your compute resources. You can increase your storage space without changing your node type.  You can increase your volume to up to 10 TB.
```

File: managed-databases/postgresql-and-mysql/how-to/create-a-database.mdx [hunk 12]
```diff
@@ -40,13 +40,15 @@ Compared to traditional database management, which requires customers to provide
         </Message>
     - Select a node type.
     - Choose your storage type and capacity. Two types of storage are available:
+        - **Basic Block Storage** - With this type, your storage is decoupled from your compute resources. You can increase your storage space without changing your node type. You can define your storage capacity by entering the desired amount in the box. You can increase your volume to up to 10 TB.
```
[the remark this answers is not in the document]
Comment (ldecarvalho-doc) [reply managed-databases/postgresql-and-mysql/how-to/create-a-database.mdx:43]:
```suggestion
        - **Basic Block Storage** - With this type, your storage is decoupled from your compute resources. You can increase your storage space without changing your node type. You can define your storage capacity by entering the desired amount in the box. The volume can be increased up to 10 TB.
```

**Review by RoRoJ — APPROVED** [review]

**Pull request merged**